bodhi/server/buildsys.py

```diff
@@ -506,7 +506,16 @@ def createTag(self, tag: str, **opts):
 
     def deleteTag(self, tagid):
         """Emulate tag deletion."""
-        del self.__tags__[tagid]
+        if isinstance(tagid, str):
+            for tid, tinfo in self.__tags__:
+                if tagid == tid:
```

Comment:
Aren't we comparing string to int here?

Reply:
This is the dummy build system, and in that case `self.__tags__` is a list of string tuples. So we should compare string with string.